front/lib/api/assistant/configuration.ts

```diff
@@ -914,6 +945,40 @@ export async function archiveAgentConfiguration(
   return updated[0] > 0;
 }
 
+export async function resurrectAgentConfiguration(
+  auth: Authenticator,
+  agentConfigurationId: string
+): Promise<boolean> {
+  const owner = auth.workspace();
+  if (!owner) {
+    throw new Error("Unexpected `auth` without `workspace`.");
+  }
+  const latestConfig = await AgentConfiguration.findOne({
+    where: {
+      sId: agentConfigurationId,
+      workspaceId: owner.id,
+    },
+    order: [["version", "DESC"]],
+    limit: 1,
+  });
+  if (!latestConfig) {
+    throw new Error("Could not find agent configuration");
+  }
+  if (latestConfig.status !== "archived") {
+    throw new Error("Agent configuration is not archived");
+  }
+  const updated = await AgentConfiguration.update(
+    { status: "active" },
+    {
+      where: {
+        id: latestConfig.id,
+      },
+    }
+  );
+
+  return updated[0] > 0;
```

Comment:
`nit` to ease reading:
```suggestion
  const affectedCount = updated;

  return affectedCount > 0;
```

Reply:
cargo culted the `archiveAssistant` fn. Will update both 👍